src/Library/demos/Status Page/main.js

```diff
@@ -0,0 +1,19 @@
+import Gtk from "gi://Gtk?version=4.0";
+
+Gtk.init();
+
+const content = workbench.builder.get_object("content");
+const entry = workbench.builder.get_object("entry_1");
+const button = workbench.builder.get_object("submit_button");
+
+button.connect("clicked", () => {
```

Comment:
As with the previous pull request, `content`, `entry` and `button` refer to objects that are not present in the BluePrint file.